Rover/AP_MotorsUGV.cpp

```diff
@@ -240,6 +242,12 @@ void AP_MotorsUGV::set_wingsail(float wingsail)
     _wingsail = constrain_float(wingsail, -100.0f, 100.0f);
 }
 
+// set mast rotation input as a value from -100 to 100
+void AP_MotorsUGV::set_mast_rotation(float wingsail)
```

Comment:
```suggestion
void AP_MotorsUGV::set_mast_rotation(float mast_rotation)
```

Reply:
Thank you for noticing this. Fixed.

Is the process in Ardupilot to squash commits before merging into master? Asking for commit messages for little fixes like this.

Reply:
Yeah, you should have one (or more) commit per directory. So you would have `Rover: something` and `SRV_Channels: something else`. There is a wiki page on how to interactive rebase. https://ardupilot.org/dev/docs/git-interactive-rebase.html#interactive-rebase-cleaning-up-commits, no real requirement to keep it tidy while its a WIP tho, feel free to wait until its working nice and then tidy up at the end.